lms/routes.py

```diff
@@ -42,8 +42,7 @@ def includeme(config):
         "blackboard_api.courses.files.list", "/api/blackboard/courses/{course_id}/files"
     )
     config.add_route(
-        "blackboard_api.files.via_url",
-        "/api/blackboard/courses/{course_id}/files/{file_id}/via_url",
+        "blackboard_api.files.via_url", "/api/blackboard/courses/{course_id}/via_url"
```

Comment:
I moved the `{file_id}` path segment out of this URL. It's not actually a file ID (it's a `document_url` in `blackboard://` format) and it's not suitable for including in the path of a URL because it has `/`'s in it. Pyramid's `route_url()` actually URL encode path segments that you pass to it but it doesn't encode `/`'s. Going with the flow here, if you pass a URL to `route_url()` in a query param instead it does get fully encoded. And I think a value like this might be more suited in a query param